packages/tools/src/eventListeners/mouse/mouseDownListener.ts

```diff
@@ -102,8 +134,30 @@ function mouseDownListener(evt: MouseEvent) {
   // Prevent CornerstoneToolsMouseMove while mouse is down
   state.element.removeEventListener('mousemove', mouseMoveListener);
 
-  const startPoints = getMouseEventPoints(evt, state.element);
-  const deltaPoints = _getDeltaPoints(startPoints, startPoints);
+  state.startPoints = getMouseEventPoints(evt, state.element);
+  state.lastPoints = _copyPoints(state.startPoints);
+
+  // First mouse down of a potential double click. So save it and start
+  // a timeout to determine a double click.
+  doubleClickState.mouseDownEvent = evt;
+
+  doubleClickState.doubleClickTimeout = setTimeout(
+    _doubleClickTimeoutHandler,
+    DOUBLE_CLICK_TOLERANCE_MS
+  );
```

Comment:
shouldn't this be the first thing right after the following?

```
if (doubleClickState.doubleClickTimeout) {
    return;
  }
```

Reply:
No I don't understand why it should be. The listeners that follow have to be added. That said I can move the `addListener` calls before the `setTimeout` call and the method will return right away regardless. However, I am still wondering why the suggestion.

Reply:
I moved the setting of the doubleClickTimeout to near the top of the method.